Tests/Common/Securities/OptionPriceModelTests.cs

```diff
@@ -332,6 +332,14 @@ public IEnumerable<HistoryRequest> GetHistoryRequirements(Security security, Dat
             public void Update(Security security, BaseData data)
             {
             }
+
+            public void ApplyDividend(Dividend dividend, bool liveMode, DataNormalizationMode mode)
```

Comment:
If these were required for compilation, that implies this change will also break existing user algorithms IF they define their own `IVolatilityModel` -- perhaps we should also have a base class `VolatilityModel` if we don't already.

Reply:
We do have `BaseVolatilityModel`. I will simplify that class by inheriting from it.